packages/react-native-performance/src/performance.ts

```diff
@@ -70,9 +109,10 @@ export const createPerformance = () => {
     }
   };
 
-  const measure = (measureName, startOrMeasureOptions = {}, endMark) => {
+  const measure = (measureName: string, startOrMeasureOptions: StartOrMeasureOptions, endMark?: string | number) => {
```

Comment:
You're removing the default options in a lot of places, are you sure you're not changing behavior?

Reply:
I added this back. There are comments for some other default options.